de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/AbstractOutgoingProjectNegotiation.java

```diff
@@ -234,7 +248,7 @@ protected void sendFileList(List<ProjectNegotiationData> projectInfos, IProgress
 
   @Override
   protected void executeCancellation() {
-    if (session.getRemoteUsers().isEmpty())
+    if (!session.isHost() || session.getRemoteUsers().isEmpty())
```

Comment:
```suggestion
    if (session.isHost() && session.getRemoteUsers().isEmpty())
```
you have to check if we are the host, otherwise the client disconnects on a canceled outgoing negotiation

Reply:
You are right, swapped that up!